Gemfile

```diff
@@ -138,6 +138,9 @@ gem 'yomu' # this gem is no longer maintained - used for custom cover letters #
 # gem 'henkei' # alternative maintained gem to yomu for cover letters # TODO: install
 gem 'htmltoword'
 
+# File path handling
+gem 'pathname'
```

Comment:
Think there is built-in functionality in ruby that should handle this without the need for an additional gem?

Reply:
Refactored to remove `pathname`